packages/react-dom/src/client/__tests__/setInnerHTML-test.js

```diff
@@ -9,60 +9,63 @@
 
 'use strict';
 
-// TODO: can we express this test with only public API?
-var setInnerHTML = require('../setInnerHTML');
-var Namespaces = require('../../shared/DOMNamespaces').Namespaces;
+const React = require('react');
+const ReactDOM = require('react-dom');
 
-describe('setInnerHTML', () => {
+describe('dangerouslySetInnerHTML', () => {
   describe('when the node has innerHTML property', () => {
     it('sets innerHTML on it', () => {
-      var node = document.createElement('div');
-      var html = '<h1>hello</h1>';
-      setInnerHTML(node, html);
-      expect(node.innerHTML).toBe(html);
+      const container = document.createElement('div');
+      const component = ReactDOM.render(
```

Comment:
Let's keep this called `node`? It's not a component.

Reply:
Sure-habit taking over from how I do this on the day-to-day. My bad!